File: src/main/java/org/example/kotlinWorkshop/java/lesson02/t1_if/t1_1/BasicIf.java
package org.example.kotlinWorkshop.java.lesson02.t1_if.t1_1;

public class BasicIf {
  public void whoIsBigger(int first, int second) {
    if(first > second) {
      System.out.println("first is bigger");
    } else if (first == second) {
      System.out.println("Numbers are equal");
    } else {
      System.out.println("second is bigger");
    }
  }
}

File: src/main/kotlin/org/example/kotlinWorkshop/kotlin/lesson02/t1_if/t1_1/BasicIf.kt
package org.example.kotlinWorkshop.kotlin.lesson02

class AABasicIf {
    fun whoIsBigger(first: Int, second: Int) {
        if (first > second) {
            println("first is bigger")
        } else if (first == second) {
            println("Numbers are equal")
        } else {
            println("second is bigger")
        }
    }
}